frontend/src/metabase/admin/settings/components/EmbeddingSettings/EmbeddingSdkSettings.tsx

```diff
@@ -173,7 +171,9 @@ export function EmbeddingSdkSettings({
           py="md"
           maw={620}
         >
-          <Text size="sm">{apiKeyBannerText}</Text>
+          <Text size="sm" data-testid="api-key-banner">
```

Comment:
is this `api-key-banner` used anywhere? I couldn't find any reference to it, if not it should be removed.

Reply:
yep, same as the other ones, I totally forgot to take these out when I was refactoring some tests